debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/NonStreamingWal2JsonMessageDecoder.java

```diff
@@ -89,11 +89,6 @@ public ChainedLogicalStreamBuilder optionsWithoutMetadata(ChainedLogicalStreamBu
             .withSlotOption("include-timestamp", 1);
     }
 
-    @Override
```

Comment:
Why is this removed? Doesn't it mean we'd get the unchanged TOAST column values again for some wal2json versions on RDS?

Reply:
@gunnarmorling We are not able to make this running correctly - see https://issues.jboss.org/browse/DBZ-1399

Reply:
Let's add the implication of this to the notes on RDS.

Reply:
@gunnarmorling Docs added